website/docs/examples/multi_client_subscriber/README.md

```diff
@@ -0,0 +1,147 @@
+---
+id: multi-client-subscriber
+title: Multi Client Subscriber
+---
+
```

Comment:
Same about adding details at the top about running locally vs on the brain and what services need to be up. Suggest to mimic the top of the filter client example